clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -439,6 +439,7 @@ get_etcd_info()
 {
   clearwater-etcdctl member list > $CURRENT_DUMP_DIR/etcd_member_list.txt
   clearwater-etcdctl cluster-health > $CURRENT_DUMP_DIR/etcd_cluster_health.txt
+  echo "When the cluster was last healthy, the members were" "$(</var/lib/clearwater-etcd/healthy_etcd_members)" >> $CURRENT_DUMP_DIR/etcd_cluster_health.txt
```

Comment:
What happens if this file isn't present? Can you have this print out a more helpful message in that case?